plugins/src/lib.rs

```diff
@@ -653,6 +662,12 @@ where
     }
 }
 
+#[derive(Clone, Debug, Deserialize)]
+pub struct Configuration {
+    #[serde(rename = "lightning-dir")]
+    pub lightning_dir: String,
+}
```

Comment:
If we keep this struct, what else should go in it?

Reply:
Here the docs https://lightning.readthedocs.io/PLUGINS.html#the-init-method and if you want to do a copy and paste https://github.com/laanwj/rust-clightning-rpc/blob/92912c8a5318624b2c9684676a0f697b4a38432e/plugin/src/commands/types.rs#L25-L51